Added example to afterSaveCommit

yosus opened the pull request:
Example to clearly show how to get $options from save

yosus committed:
Added example to afterSaveCommit

Example to clearly show how to get $options from save

ndm2 commented:
Is there a reason that the example would go specifically there, and _only_ there, given that there's many other callbacks with options too?

IMHO it would be more suited in the [Saving Data](https://book.cakephp.org/4/en/orm/saving-data.html#saving-entities) page, and/or in a dedicated subsection under [Lifecycle Callbacks](https://book.cakephp.org/4/en/orm/table-objects.html#lifecycle-callbacks).

Arhell reviewed (commented):
cc @markstory

markstory commented on en/orm/table-objects.rst:
```diff
@@ -304,6 +304,18 @@ saves where database operations are implicitly committed. The event is triggered
 only for the primary table on which ``save()`` is directly called. The event is
 not triggered if a transaction is started before calling save.
```
```suggestion
not triggered if a transaction is started before calling save::
```

This line needs to end in `::` or the code block won't be formatted.

markstory reviewed (commented):
I agree with @ndm2. Accessing options is common to all lifecycle callbacks. I will can relocate the section post merge though if you address the formatting problem.

yosus committed:
Moved example to Lifecycle Callbacks

Thanks. I was contemplating where is the best place to put the example so `$options` can be linked to `save`. 
I have made the changes and also added `afterSave`, so users will know `$options` are linked between callback-events.

yosus reviewed (commented):
moved example to `Lifecycle Callbacks`
